db/migrate/20170811203356_suggestion.rb

```diff
@@ -0,0 +1,14 @@
+class Suggestion < ActiveRecord::Migration[5.1]
+  def change
+    create_table :suggestions do |t|
+      t.references :user, foreign_key: true
+      t.string :author, null: false
+      t.string :link, null: false
+      t.string :title, null: false
+      t.string :publisher, null: false
+      t.string :year, null: false
+      t.string :editorial, null: false
+      t.integer :price
```

Comment:
The price must be a float accordingly to the card.

Note: using floats to store money is a very bad practice... but lets use it anyway for now. If we have time we will change it with something more robust.